sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -26,13 +27,13 @@ export const ContentPlacementContainerStyles = css`
         border-bottom: none;
     }
 
-    @media screen and (min-width: 700px) {
+    @media screen and (min-width: ${siteBreakpoints[1]}) {
```

Comment:
Readability: see comments above above.

Reply:
Reverting this to fixed numbers for now.  Proposal coming forth on this.  See Chris' comment below.